cpp/include/cuspatial/shapefile_reader.hpp

```diff
@@ -23,6 +23,11 @@
 
 namespace cuspatial {
 
+typedef enum class winding_order : bool {
+  CLOCKWISE,
+  COUNTER_CLOCKWISE  // or "anticlockwise" or "widdershins", if you prefer
```

Comment:
I'm not sure if this is a stale comment, or do you mean to say this is sometimes referred to as "anticlockwise"/"widdershins" in other literature? We should remove this in the former, or change the comment for the latter.
```suggestion
  COUNTER_CLOCKWISE
```

Reply:
I inserted that comment purely as a linguistic joke when I originally made the suggestion. Please delete.

Reply:
Aww. Done! :D